lesson2/sum mid and num.py

```diff
@@ -0,0 +1,18 @@
+if __name__ == '__main__':
+
+    summ = 0
+    i=float(input('Input number end press Enter: '))
+    num = []
+    counter=0
+    while i!=0:
+       counter+=1
+       num.append(i)
+       i = float(input('Input number end press Enter: '))
+    for a in range(len(num)):
```

Comment:
The same as above. You don't need to store all numbers, just use 2 variables - sum and quantity

Reply:
Done